src/StoragelessSession/Http/SessionMiddleware.php

```diff
@@ -0,0 +1,144 @@
+<?php
+namespace StoragelessSession\Http;
+
+use DateTime;
+use Zend\Stratigility\MiddlewareInterface;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+use Lcobucci\JWT\Token;
+use Lcobucci\JWT\Signer;
+use Lcobucci\JWT\Builder;
+use StoragelessSession\Session\Data;
+use Lcobucci\JWT\Parser;
+use Lcobucci\JWT\ValidationData;
+use Lcobucci\JWT\Signer\Key;
+
+final class SessionMiddleware implements MiddlewareInterface
```

Comment:
Yeah, no. I don't code against conventions, nor will do that in future.

That's why I simply prefer having a dependency to stratigility, so that BC breaks can be tracked, and the BC policy depends on the interface future changes.